.github/workflows/nightly.yml

```diff
@@ -9,6 +9,7 @@ jobs:
   publish:
     name: 'Publish'
     runs-on: ubuntu-latest
+    if: github.repository == 'strapi/strapi'
```

Comment:
What do you think about using `github.repository_owner == 'strapi'` instead? Makes the job a bit more general and easier to share across repos (if ever needed one day).